scripts/gen_vimdoc.py

```diff
@@ -70,13 +70,10 @@
     ]
 }
 
-fmt_vimhelp = False  # HACK
 text_width = 78
 script_path = os.path.abspath(__file__)
 base_dir = os.path.dirname(os.path.dirname(script_path))
-out_dir = os.path.join(base_dir, 'tmp-{target}-doc')
```

Comment:
Even though it's only used in one place, it still makes sense for it to live next to related declarations.